### PR TITLE
Static batch test

### PR DESCRIPTION
**Pull request by martyn**
New automated search for g/d combinations (like 2d-measure-accuracy).

Add more hyperparams to control batch norm.

### CODE REVIEW

File: hypergan/generators/common.py [hunk 1]
```diff
@@ -3,12 +3,20 @@
 import hyperchamber as hc
 from hypergan.util.hc_tf import *
 
+def repeating_block(net, config, activation, batch_size,id,name, resize=None, output_channels=None, stride=2, noise_shape=None, dtype=tf.float32,filter=3, batch_norm=None, sigmoid_gate=None, reshaped_z_proj=None):
```
Comment (martyn) [inline hypergan/generators/common.py]: This needs some refactoring

File: hypergan/generators/resize_conv_generator.py [hunk 2]
```diff
@@ -28,11 +33,15 @@ def config(
...
     selector.set("depth_reduction", depth_reduction) # Divides our depth by this amount every time we go up in size
     selector.set('layer_filter', layer_filter) #Add information to g
-    selector.set('layer_regularizer', batch_norm_1)
+    selector.set('layer_regularizer', layer_regularizer)
     selector.set('block', block)
+    selector.set('block_repeat_count', block_repeat_count)
     selector.set('resize_image_type', resize_image_type)
     selector.set('sigmoid_gate', sigmoid_gate)
 
+    selector.set('orthogonal_initializer_gain', orthogonal_initializer_gain)
```
Comment (martyn) [inline hypergan/generators/resize_conv_generator.py]: Need to update docs.  I'll add to pivotal.

File: hypergan/util/hc_tf.py [hunk 3]
```diff
@@ -90,11 +90,11 @@ def residual_block(result, activation, batch_size,id,name):
     return left+right
 
 #TODO move this somewhere?  Used by graph creation
-def block_conv(result, activation, batch_size,id,name, resize=None, output_channels=None, stride=2, noise_shape=None, dtype=tf.float32,filter=3, batch_norm=None, sigmoid_gate=None, reshaped_z_proj=None, gain=1.0):
+def block_conv(result, config, activation, batch_size,id,name, resize=None, output_channels=None, stride=2, noise_shape=None, dtype=tf.float32,filter=3, batch_norm=None, sigmoid_gate=None, reshaped_z_proj=None, gain=1.0):
     size = int(result.get_shape()[-1])
     result = activation(result)
     if(batch_norm is not None):
-        result = batch_norm(batch_size, name=name+'bn')(result)
+        result = batch_norm(batch_size, momentum=config.batch_norm_momentum, epsilon=config.batch_norm_epsilon, name=name+'bn')(result)
```
Comment (martyn) [inline hypergan/util/hc_tf.py]: These arguments are telescoping too much, like above.
